src/main/java/com/synopsys/integration/detect/lifecycle/shutdown/ExceptionUtility.java

```diff
@@ -15,7 +15,7 @@
 
 public class ExceptionUtility {
     private final Logger logger = LoggerFactory.getLogger(this.getClass());
-    private static final String BLACKDUCK_ERROR_MESSAGE = "An unrecoverable error occurred - most likely this is due to your environment and/or configuration. Please double check the Detect documentation: https://detect.synopsys.com/doc/";
+    private static final String BLACKDUCK_ERROR_MESSAGE = "An unrecoverable error occurred - most likely this is due to your environment and/or configuration. Please double check the Detect documentation: https://sig-product-docs.synopsys.com/bundle/integrations-detect/page/introduction.html";
```

Comment:
" occurred. It is most likely due to the environment and/or the configuration. "?

Reply:
Tweaked. Shrunk the message a bit at the same time. Thank you.